debian/snapd.postrm

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+if [ "$1" = "purge" ]; then
+   # FIXME: should we try to remove all snaps, mount points, services,
+   #        mount units that got created as well?
+   rm -f /var/lib/snapd/state.json
```

Comment:
can we echo some kind of warnings? maybe listing the units we likely are orphaning?